backend/src/graphql/resolver/ContributionLinkResolver.ts

```diff
@@ -39,35 +40,30 @@ export class ContributionLinkResolver {
     }: ContributionLinkArgs,
   ): Promise<ContributionLink> {
     isStartEndDateValid(validFrom, validTo)
+    // TODO: this should be enforced by the schema.
```

Comment:
It is. If you call the API without `name`, you get a schema error.
The test is the same as `name.length === 0` and there for obsolete as we test that length must be greater than `x`

Reply:
![image](https://user-images.githubusercontent.com/1238238/217111855-73e26f85-c64f-466a-b6a8-48660d638d79.png)

the whole thing requires refactoring - lets talk about solutions

Reply:
Yes, we can do this. My point is, that the test `(!name)` can only be true if the name is an empty string and therefore useless, as we test later for a minimum length

Reply:
How can we localize error messages thrown by this validator?

Reply:
> How can we localize error messages thrown by this validator?

We do not localize Error messages - but I am sure there is a way.

Reply:
I see your point and will remove the check & test

Reply:
Solved with your suggestion and created Issue #2672 for the validator part.